configparser.go

```diff
@@ -255,9 +252,8 @@ func (p *ConfigParser) ParseReader(in io.Reader) error {
 				// multiline prefixes or it is an empty line which is not allowed within values,
 				// then it counts as the value parsing is finished and it can be added
 				// to the current section.
-				if err := curSect.Add(key, value); err != nil {
-					return fmt.Errorf("failed to add %q = %q: %w", key, value, err)
-				}
+				// Add never returns an error.
+				_ = curSect.Add(key, value)
```

Comment:
This code is nearly 12 years old, and has remained mostly untouched since it was first written 😄 

I'll definitely fix `.Add()` not returning an error in a `/v2` 😄 along with some other things like error wrapping.